tests/rptest/tests/rpk_start_test.py

```diff
@@ -13,6 +13,7 @@
 from rptest.clients.rpk_remote import RpkRemoteTool
 from rptest.services.redpanda import RedpandaService
 
+import concurrent.futures
```

Comment:
I can't see where it is used

Reply:
Ah oops, I opted to reuse `_for_nodes()` instead